dags/probe_scraper.py

```diff
@@ -394,71 +352,11 @@ def choose_branch(self, context):
 
     probe_scraper >> delay_python_task
 
-    image_tag = Variable.get("lookml_generator_release_str")
-    if image_tag is None:
-        image_tag = DEFAULT_LOOKML_GENERATOR_IMAGE_VERSION
-
-    lookml_generator_prod = GKEPodOperator(
-        owner="ascholtz@mozilla.com",
-        email=[
-            "ascholtz@mozilla.com",
-            "dataops+alerts@mozilla.com",
-            "telemetry-alerts@mozilla.com",
-        ],
-        task_id="lookml_generator",
-        name="lookml-generator-1",
-        image="gcr.io/moz-fx-data-airflow-prod-88e0/lookml-generator:" + image_tag,
-        startup_timeout_seconds=500,
-        dag=dag,
-        env_vars={
-            "HUB_REPO_URL": "git@github.com:mozilla/looker-hub.git",
-            "HUB_BRANCH_SOURCE": "base",
-            "HUB_BRANCH_PUBLISH": "main",
-            "SPOKE_REPO_URL": "git@github.com:mozilla/looker-spoke-default.git",
-            "SPOKE_BRANCH_PUBLISH": "main",
-            "LOOKER_INSTANCE_URI": "https://mozilla.cloud.looker.com",
-            "UPDATE_SPOKE_BRANCHES": "true",
-        },
-        secrets=[
-            looker_repos_secret_git_ssh_key_b64,
-            looker_api_client_id_prod,
-            looker_api_client_secret_prod,
-            dataops_looker_github_secret_access_token,
-        ],
-        **airflow_gke_prod_kwargs,
+    # trigger lookml generation
+    trigger_looker = TriggerDagRunOperator(
+        task_id="lookml_generator_staging", trigger_dag_id="looker"
```

Comment:
I think the task id should be updated as well
```suggestion
        task_id="trigger_lookml_generator", trigger_dag_id="looker"
```